nuxeo-drive-client/nxdrive/manager.py

```diff
@@ -1063,7 +1063,7 @@ def get_proxies(self, server_url):
         elif proxy_settings.config == 'System':
             proxies = self.retreive_system_proxies(server_url)
         elif proxy_settings.config == 'Automatic':
-            if server_url not in self.proxies:
+            if server_url not in self.proxies or self.proxies[server_url] is None:
```

Comment:
You could use `self.proxies.get(server_url) is None` to prevent double checks.

Reply:
done.